src/utils.tsx

```diff
@@ -236,4 +236,6 @@ export const extractDailyOracles = (tokenName: string, dailyMetricsResponse: any
   }
 
   return dailyOracles;
-};
\ No newline at end of file
+};
+
+export const parseBigInt = (str: string) => Number(str.slice(0, -1))
```

Comment:
I think it can be trouble if the value is way too large:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/Number#using_number_to_convert_a_bigint_to_a_number

Why not replace `Number()` with `BigInt()`?

Reply:
all the other values we use in our calculations are numbers instead of bigints
making this bigInt would introduce type issues from performing arithmetic operations between bigint and number

Reply:
all the other values we use in our calculations are numbers instead of bigints
making this bigInt would introduce type issues from performing arithmetic operations between bigint and number

Reply:
all the other values we use in our calculations are numbers instead of bigints
making this bigInt would introduce type issues from performing arithmetic operations between bigint and number

Reply:
Interesting..........I think we should use `BigInt` constructor for type coercion where possible. It's possible to get issues with `Number()` coercion as pointed out by the link I shared. 

@toliaqat @frazarshad Do you think we should create an issue for this? We can add it to the backlog to update all the instances where we should use BigInt? Or it's not required?